### ISSUE TITLE
method setMaxDistance need DateTime

### ISSUE THREAD

**Issue by amaille** [eymengunay/php-passbook#46]
In the Pass class, i can't set the max distance. I don't understand why max distance needs to be DateTime. 

Thanks.

```  
    /**
     * {@inheritdoc}
     */
    public function setMaxDistance(\DateTime $maxDistance)
    {
        $this->maxDistance = $maxDistance;
        return $this;
    }

```

**Comment by g-** [comment]
Hmm. That looks like a pretty clear bug. Let me have a look.

**Comment by g-** [comment]
Just released Version 1.2.1 with a fix for this and I changed the composer requirement for php back to 5.3 just in case you were using that version.
